janusgraph-codepipelines-ci/README.md

```diff
@@ -72,44 +72,68 @@ per region per account. If you need more, you can
 Here is an example of a file that defines two pipelines with parallel build actions.
 
 ```yaml
+defaultComputeImage: stephenreed/jenkins-java8-maven-git:latest
```

Comment:
I was able to run tests using the `maven:3-jdk-8` image. Apparently git isn't required ... maybe code is checked out in pipeline external to compute image? I'd prefer this if possible because it's an official image and Apache licensed (I don't see license information on the `stephenreed/jenkins-java8-maven-git` image).